/*
 * data_generator.h
 *
 *  Created on: Dec 15, 2021
 *      Author: steelph0enix
 */

#ifndef INC_APP_DATA_GENERATOR_H_
#define INC_APP_DATA_GENERATOR_H_

#include <stdint.h>
#include <stddef.h>

size_t generateData(uint8_t* dataBuffer, size_t bufferSize);

#endif /* INC_APP_DATA_GENERATOR_H_ */
